content/en/docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md

```diff
@@ -78,9 +78,8 @@ unless the Pod's grace period expires. For more details, see
 [Pod Lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/).
 
 {{< note >}}
-Kubernetes only sends the preStop event when a Pod is *terminated*.
+Kubernetes only sends the preStop event when a Pod or a container in the Pod is *terminated*.
 This means that the preStop hook is not invoked when the Pod is *completed*. 
-This limitation is tracked in [issue #55087](https://github.com/kubernetes/kubernetes/issues/55807).
```

Comment:
perhaps instead of this say - there are cases when it will not be called. Maybe add to the documentaiton that was added as a result of the issue resolution.

Reply:
Sorry, I don't understand what you say(maybe due to my poor English skill)
You're suggesting deleting L83 and adding summary of the limitation(issue [#55087](https://github.com/kubernetes/kubernetes/issues/55807) )?

Reply:
@SergeyKanzhelev sorry, could you please check and reply to my comment? thanks.

Reply:
I guess @SergeyKanzhelev think there still are some cases preStop hook is not called, and if so it is better to describe here.
But I am not sure what we need to write here.
It would be good enough to just delete the limitation which has been already solved like this pull request.

Reply:
Following the discussions in the issue mentioned here (https://github.com/kubernetes/kubernetes/issues/55807#issuecomment-463922285), more detail descpription was added as a follow-up (https://github.com/kubernetes/website/pull/12498).

The `preStop` hook section of https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#container-hooks provides the detail information.

How about mentioning the above page for more detail instead of just dropping this line?

Reply:
@amotoki @SergeyKanzhelev 

Is it fine to replace the above line with
```
A more detailed description of preStop hook can be found in [Container Hooks](/docs/concepts/containers/container-lifecycle-hooks/#container-hooks).
```

Reply:
lgtm
@YuikoTakada Could you update this pull request based on @amotoki's suggetion?